source/imports/value-objects/nin.js

```diff
@@ -0,0 +1,10 @@
+const NIN = Space.domain.ValueObject.extend('NIN', {
```

Comment:
Never heard about that, would be good to have a comment in the code with a link for reference